src/Insurance/App/Transformers/PayerDMETransformer.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Insurance\App\Transformers;
+
+use Flugg\Responder\Transformers\Transformer;
+use Lightit\Shared\Domain\Models\PayersDMEProvider;
+
+class PayerDMETransformer extends Transformer
+{
+    protected $relations = [
+        'dmeProvider' => DMEProviderTransformer::class,
+    ];
+
+    public function transform(PayersDMEProvider $payerDMEProvider): array
+    {
+        return [
+            'id' => $payerDMEProvider->id,
+            'dmeProvider' => (new DMEProviderTransformer())->transform($payerDMEProvider->dmeProvider),
```

Comment:
Is this needed if it is specified in `$relations`? Or maybe we can use `$load`?

Reply:
wasn't working so just added this to solve, it should be working by just adding the protected relation, but it didn't work in my local.

Reply:
I think for it to work you'd need to add `->with('dmeProvider')` after the `->success` in the controller, or replace `relations` with `load`.